runtime/compiler/env/j9method.cpp

```diff
@@ -7454,6 +7454,91 @@ static TR::DataType typeFromSig(char sig)
    return TR::NoType;
    }
 
+char* sigForPrimitiveOrVoid(TR_OpaqueClassBlock* clazz)
```

Comment:
can you add `static` keyword so that it's clearer the method is only accessible in this file?